Readme.md

```diff
@@ -1,24 +1,55 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128533319/13.1.4%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3583)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to apply a filter to a column on the client side
+# GridView for ASP.NET Web Forms - How to apply a filter to a column on the client side
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3583/)**
 <!-- run online end -->
 
+This example demonstrates how to use a text string from the external text box to filter a grid column on the client.
```

Comment:
> This example demonstrates how to use a text string from **the** external text box to filter a grid column on the client.

Of what external text box do we speak? We do not know anything of it just yet. This entire introductory sentence speaks of general and abstract things, like it should. Therefore, we should not make an exception of this external text box and make it indefinite.

```suggestion
This example demonstrates how to use a text string from an external text box to filter a grid column on the client.
```